orc-rt/tools/orc-executor/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(orc-executor orc-executor.cpp)
+target_link_libraries(orc-executor PRIVATE orc-rt-executor)
+target_include_directories(orc-executor PRIVATE ../../include)
```

Comment:
This should be some target providing it. Though `orc-rt-executor` public depends on `orc-rt-headers`, so it should actually be unnecessary.

Reply:
I'd appreciate any guidance you can give here?

For now `orc-rt-executor` is primarily a test-tool: We want it to run regression tests for developers of the ORC runtime (as opposed to clients).

Some clients might also want to be able to install it as a useful tool in its own right. I'm not sure how to best express this in CMake. Maybe another component? `OrcRT_ExtraTools`?

Reply:
I'd try just removing this line. Generally, one should use `target_*` to add either:

- paths, flags, or options for *this* target
- use targets to provide anything "external" to the target itself

The issue is that, to me, using `../` smells of "someone else's path detail" that should come through a target's usage requirement. It *looks* like it flows through `orc-rt-executor` already, but it's just as easy to build-confirm the suspicion.

I think a `tools` component makes sense (feel free to namespace however you want).

Reply:
Thanks -- I've dropped the target_include_headers line here in a recent commit.